src/pages/AccountSelector.tsx

```diff
@@ -0,0 +1,73 @@
+import type { InjectedAccountWithMeta } from "@polkadot/extension-inject/types";
+import { ContinueButton } from "../components/buttons/ContinueButton";
+
+type AccountSelectorProps = {
+  accounts: InjectedAccountWithMeta[];
+  selectedAccount: InjectedAccountWithMeta | null;
+  onSelectAccount: (account: InjectedAccountWithMeta) => void;
+  onContinue: () => void;
+};
+
+const AccountSelector = ({
```

Comment:
UX: I don't think this should be on a separate page, this page should work even if you didn't connect your wallet, so it's a nice showcase. It should just throw when you're trying to do anything and you did not select your account.